### PR TITLE
Setup redux folder structure

### PR DESCRIPTION
**Pull request by h01m3s**

### CODE REVIEW

File: client/src/redux/store.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { createStore, applyMiddleware } from 'redux';
+import thunk from 'redux-thunk';
+import rootReducer from './reducers/';
+
+let store = createStore(rootReducer, applyMiddleware(thunk));
```
Comment (chengmuy) [inline client/src/redux/store.js]: Can we add in the code that enables Redux DevTools to work?

`const store = createStore(rootReducer, window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__());`

File: client/src/index.js [hunk 2]
```diff
@@ -3,11 +3,14 @@ import ReactDOM from 'react-dom';
 import './styles/index.css';
 import App from './components/App';
 import * as serviceWorker from './serviceWorker';
+import { Provider } from 'react-redux';
+import store from './redux/store';
 
 ReactDOM.render(
-  <React.StrictMode>
+  <Provider store={store}>
```
Comment (chengmuy) [inline client/src/index.js]: Let's keep strict mode turned on as well?

File: client/src/redux/store.js (hunk 1 above)
Comment (chengmuy) [inline client/src/redux/store.js]: Can we add in the code that enables Redux DevTools to work?

(from a different project:)
`const store = createStore(rootReducer, window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__());`

File: client/src/redux/actions/currentProduct.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+var currentProduct = (product) => ({
```
Comment (chengmuy) [inline client/src/redux/actions/currentProduct.js]: can we name as "setCurrent" or "setCurrentProduct", with type "SET_CURRENT" or "SET_CURRENT_PRODUCT"?

I prefer setCurrent/SET_CURRENT

**Comment by h01m3s** [comment]
[Redux structure + setup](https://trello.com/c/vOLjZSCs/51-redux-structure-setup)

**Pull request merged**